src/configs/FirebaseConfig.js

```diff
@@ -13,6 +14,7 @@ import {
   POST_CHECK_FEED_EXPIRED,
   POST_CHECK_FEED_NOT_FOUND
 } from '../utils/constants';
+import {getMyProfile} from '../service/profile';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `getMyProfile` is added here. Ensure that this function is properly defined and exported in the `../service/profile` module.
```diff
+ import {getMyProfile} from '../service/profile';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->